draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2660,12 +2670,18 @@ http://example.com/hls/35cdc008/audio/index.m3u8\n”,
 	Mandatory: No; default is "false".  The query part of the URI
          should be dropped before comparison with the given pattern.
 	</t>
+        <t hangText="Name: url-type">
+        <vspace blankLines="1"/>
+        Description: Type of URLs to match.
+        <vspace blankLines="1"/>
+        Value: URL Type as defined in  <xref target="url-types"/>.
```

Comment:
Not sure if this will matter for rendering, but there is an extra space after "in".